DependencyInjection/CmfSeoExtension.php

```diff
@@ -78,6 +78,11 @@ public function load(array $configs, ContainerBuilder $container)
             $sonataBundles[] = 'SonataDoctrineORMBundle';
         }
 
+        $container->setParameter($this->getAlias().'.form_mode_orm',
```

Comment:
we can't use the backend_type_* parameters because they must only be set if true (the doctrine compiler pass only checks if the parameter exists, not the value of the parameter). and anyways, we want this logic in case somebody enabled both backends.